src/osyris/plot/parser.py

```diff
@@ -1,14 +1,22 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright (c) 2022 Osyris contributors (https://github.com/nvaytet/osyris)
 
-from matplotlib.colors import LogNorm, Normalize
+from matplotlib.colors import LogNorm, Normalize, SymLogNorm
 
 
 def get_norm(norm=None, vmin=None, vmax=None):
-    if norm == "log":
-        return LogNorm(vmin=vmin, vmax=vmax)
-    else:
+    if norm is None:
         return Normalize(vmin=vmin, vmax=vmax)
+    if isinstance(norm, str):
+        norm_lowercase = norm.lower()
+        if norm_lowercase == "log":
+            return LogNorm(vmin=vmin, vmax=vmax)
+        elif norm_lowercase == "symlog":
+            return SymLogNorm(linthresh=1e-2, vmin=vmin, vmax=vmax, base=10)
+        elif norm_lowercase == "linear":
+            return Normalize(vmin=vmin, vmax=vmax)
```

Comment:
I think you should raise an error here if there is no match for the string, otherwise it will just return `None`

Reply:
Nice catch.